pkgs/development/python-modules/pyutilib/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, nose
+, six
+}:
+
+buildPythonPackage rec {
+  pname = "PyUtilib";
+  name = "${pname}-${version}";
+  version = "5.6.2";
+
+  propagatedBuildInputs = [ nose six ];
```

Comment:
`nose` should go into `checkInputs`

Reply:
Moving `nose` to `checkInputs` make the build fail:
```
Collecting nose (from PyUtilib==5.6.2)
  Could not find a version that satisfies the requirement nose (from PyUtilib==5.6.2) (from versions: )
No matching distribution found for nose (from PyUtilib==5.6.2)
builder for '/nix/store/z2b5nxy01cdns11kk8dhp1idraars3lc-python3.6-PyUtilib-5.6.2.drv' failed with exit code 1
error: build of '/nix/store/z2b5nxy01cdns11kk8dhp1idraars3lc-python3.6-PyUtilib-5.6.2.drv' failed
```

Reply:
Indeed, that's a mistake on their side probably: https://github.com/PyUtilib/pyutilib/blob/master/setup.py#L42
Please also note the additional requirements for Python 2.